Dockerfile.IntelNetSecVSP.rhel

```diff
@@ -23,6 +23,7 @@ COPY --from=builder /workspace/bin/vsp-intel-netsec.${TARGETARCH} /vsp-intel-net
 
 RUN yum update -y \
     && yum install -y \
+       ethtool \
```

Comment:
yes, makes (probably) sense.

But why is this done? The commit message should explain that. Even if you just have a fuzzy feeling that you prefer having ethtool over not having it, then that should be recorded. You don't need strong evidence why this is absolutely necessary (because there is also no strong evidence why this shouldn't be there). But in that case, just say exactly that.

The benefit is if we tomorrow review the container (maybe to reduce the container size), we might want to know why this package was added in the first place.

Reply:
I updated the PR description.
```
This PR updates the PTL VSP container adding Ethtool for avoiding warnings that Ethtool is not installed, this is because the PCI utilities uses ethtool to check the capabilities of the interfaces. 
```